spec/system/index_spec.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe "Index page", type: :feature do
```

Comment:
This could also go within a `feature` directory. https://relishapp.com/rspec/rspec-rails/v/5-0/docs/feature-specs/feature-spec

Reply:
Cool, makes sense, I'm updating in Issue 3 PR